projects/wvr-elements/src/lib/shared/utility/bootstrap.utility.ts

```diff
@@ -2,10 +2,12 @@ import { Type } from '@angular/core';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 import { AppConfig, APP_CONFIG } from '../config';
 
+/** Provides the current script. */
+const componentScript = document.currentScript;
```

Comment:
I think it makes sense for this to be within obtainConfigPath scope now that we know the previous issue was from angular cli serve.

Reply:
`componentScript` moved within the scope of `obtainConfigPath`.

Reply:
Thanks. Can you fix the indentation?

Reply:
Tests at local end passing however the build at Travis end is in progress.